Index: src/LambdaPractice/MyFrame.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package LambdaPractice;\r\n\r\nimport javax.swing.*;\r\n\r\npublic class MyFrame extends JFrame {\r\n\r\n    JButton myButton = new JButton(\"MY BUTTON 1\");\r\n    JButton myButton2 = new JButton(\"MY BUTTON 2\");\r\n\r\n    MyFrame(){\r\n\r\n        myButton.setBounds(100,100,200,100);\r\n        myButton.addActionListener(\r\n            (e) -> System.out.println(\"You clicked the first button\"));\r\n\r\n        myButton2.setBounds(100,200,200,100);\r\n        myButton2.addActionListener(\r\n            (e) ->System.out.println(\"You clicked the second button\"));\r\n\r\n        this.add(myButton);\r\n        this.add(myButton2);\r\n        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);\r\n        this.setSize(420,420);\r\n        this.setLayout(null);\r\n        this.setVisible(true);\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/LambdaPractice/MyFrame.java b/src/LambdaPractice/MyFrame.java
--- a/src/LambdaPractice/MyFrame.java	
+++ b/src/LambdaPractice/MyFrame.java	
@@ -9,11 +9,11 @@
 
     MyFrame(){
 
-        myButton.setBounds(100,100,200,100);
+        myButton.setBounds(50,50,100,50);
         myButton.addActionListener(
             (e) -> System.out.println("You clicked the first button"));
 
-        myButton2.setBounds(100,200,200,100);
+        myButton2.setBounds(50,100,100,50);
         myButton2.addActionListener(
             (e) ->System.out.println("You clicked the second button"));
 
